src/Thanks.php

```diff
@@ -39,4 +44,22 @@ public function getCommands()
             new Command\ThanksCommand(),
         ];
     }
+
+    public function displayReminder(Script\Event $event)
+    {
+        $love = '\\' === DIRECTORY_SEPARATOR ? 'love' : '💖 ';
+        $star = '\\' === DIRECTORY_SEPARATOR ? 'star' : '⭐ ';
+
+        $this->io->writeError('');
+        $this->io->writeError('What about running <comment>composer thanks</> now?');
+        $this->io->writeError(sprintf('This will spread some %s by sending a %s to the GitHub repositories of your fellow package maintainers.', $love, $star));
```

Comment:
Other suggestions welcome.
I'd just like to have a sentence that convey the idea that there are human being behind the code (you thank people, not code.)

Reply:
Maybe `by sending a {star} to the Github repositories of the awesome people that maintain the packages you are using`? 😄

Reply:
I actually like the sentence now - it sounds pretty cool to me :)